cmd/sd/internal/commands/config/path/path.go

```
@@ -0,0 +1,28 @@
+	"github.com/Southclaws/storyden/cmd/sd/internal/help"
+)
+
+type PathCommand *cobra.Command
+
+func New(store *config.Store) PathCommand {
+	command := &cobra.Command{
+		Use:   "path",
+		Short: "Print the sd config file path",
+		Args:  cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			fmt.Fprintln(cmd.OutOrStdout(), store.Path())
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)